litestar/channels/redis.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import annotations
```

Comment:
This file should be renamed, otherwise there will be import errors. 

`redis_channels_backend.py`

Reply:
That's our established pattern. We have `stores.redis`, `contrib.sqlalchemy`, `contrib.jinja`, etc.